prep-javascript-variables/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'Alan Ho';
```

Comment:
make sure to declare all of the variables using `const` except for the never variable.